src/pint/modelutils.py

```diff
@@ -2,9 +2,17 @@
 
 
 def convert_to_equatorial(model):
-    """Converts pulsar's PulsarEcliptic coordinates to ICRS,
-    adds the AstrometryEquatorial model component with the
-    correct values, and removes AstrometryEcliptic component.
+    """Converts Astrometry model component, Ecliptic to Equatorial
+
+      Parameters
```

Comment:
I am not sure if this will cause any indentation error when doc building. I think the best would be like
```
"""
Convert Astro.....

Parameters
---------------
model: `pint.models.TimingModel` object
    Timing model with astrometry Ecl.....

Return
---------
    New timing model with eqtor
"""
```

Reply:
The mark down did not line up. But if you remove the ``` magic will happen.

Reply:
I don't think I understand this comment. I ran black, but it sounds like you're talking about something else.

Reply:
I've been trying to be consistent using numpydoc style:
https://numpydoc.readthedocs.io/en/latest/example.html#source

Reply:
Sorry for lacking context, there are two parts. 

1. It is the indentation of the docstrings.  
```
"""
  Parameters
  ---------------
"""
```
 vs.

```
"""
Parameters
---------------
"""
```
2. It is about the github comment box. 
if I write the docstring using numpy styles here in this comment. It will look like: 

"""
Convert Astro.....

Parameters
---------------
model: `pint.models.TimingModel` object
    Timing model with astrometry Ecl.....

Return
---------
    New timing model with eqtor
"""
However, if we use "```" outside of """, which indicates a code block, it will look like 
```
"""
Convert Astro.....

Parameters
---------------
model: `pint.models.TimingModel` object
    Timing model with astrometry Ecl.....

Return
---------
    New timing model with eqtor
"""
```

Reply:
Hmmm I think I forgot about this. Let me double-check to see if it's fixed...

Reply:
Ok, I think I've fixed this in the latest commit. Doesn't look like it's showing up quite yet...